File: Java_Design-Patterns/src/facade/PointCarte.java
package facade;

public class PointCarte {
    private double x, y;

    public PointCarte(double x, double y) {
        this.x = x;
        this.y = y;
    }

    public void move(double dx, double dy) {
        this.x += dx;
        this.y += dy;
    }

    @Override
    public String toString() {
        return "(" + x + "," + y + ")";
    }

    public double getX() {
        return x;
    }

    public double getY() {
        return y;
    }
}

File: Python_Design-Patterns/facade/point_carte.py
class PointCarte:

    def __init__(self, x, y):
        self.x = x
        self.y = y

    def move(self, dx, dy):
        self.x = dx
        self.y = dy

    def __str__(self):
        return f'({self.x},{self.y})'

    @property
    def x(self):
        return self.x

    @property
    def y(self):
        return self.y